### DISCUSSION TITLE
Format shortcut

### DISCUSSION THREAD

**Discussion by RemiP12** [GorvGoyl/Notion-Boost-browser-extension#175] [Ideas]
Hi! 
Thank you so much for your work, the sticky sidebar in particular is incredibly useful!

Tldr: I'd like to be able to assign a keystroke to a format, for example `Alt+T` = `Ctrl+B` + `Ctrl+E` + blue text + yellow background

I'm writing a pretty lengthy Wiki, so I'm looking for shortcuts. For example, I like to format my inline code in the default code format (Ctrl+E), and differentiate it from written shortcut keys and paths by the color. So I would have something like:
![image](https://user-images.githubusercontent.com/98538657/228954277-9dafee25-56e8-4d39-9c30-a61305e4069e.png)
Formatting in blue code requires to do `Ctrl+E` + `Ctrl+Shift+H` (`Ctrl+Shift+H` is actually very uncomfortable to do for a single hand, and this is assuming that blue is the last colour I've formatted in), so ideally I'd like to be able to manually assign a shortcut to a specific format instead of using the format bar or multiple shortcuts. 

Thank you, let me know if I can help!
